dspace/config/modules/workflow.cfg

```diff
@@ -8,7 +8,7 @@
 # Possible values:
 #   originalworkflow = Traditional DSpace Workflow
 #   xmlworkflow = New (as of 1.8.0) Configurable Reviewer Workflow
-workflow.framework=originalworkflow
+workflow.workflow.framework=originalworkflow
```

Comment:
Yuck. Looks like we need to correct the code that is asking for this config, so that the name can be kept as `workflow.framework` instead of prefixing it twice as `workflow.workflow.framework`.

This isn't a fault of your PR. It's just that your PR made me realize we need to fix the code which uses the `workflow.framework` config to avoid this double prefixing.  I'll work on a separate PR to fix this.